src/containers/WelcomePage/WelcomePage.jsx

```diff
@@ -163,6 +255,86 @@ export class WelcomePage extends Component {
   }
 }
 
+function mapSearchToFrontPageStructure(data, t) {
```

Comment:
Jeg har ikke sett mye på denne, men er denne funksjonen nødvendig, eventuelt veldig kompleks? Merker jeg sliter med å forstå alt som skjer.

Reply:
Mulig å få til noe ala det som skjer i `searchHelpers/convertResult`?

Reply:
Nei ikke helt det samme... den må gruppere resultatet og lage ny struktur. Og ta vekk eventuelle duplikater.

Reply:
Hvorfor ikke lage en egen graphql-spørring for framsiden? Det er vel hele poenget med Graphql, at man skal hente akkurat den dataen man trenger